<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="swirGroupBox">
      <property name="title">
       <string>短波相机参数</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>对比度</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelGain">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>150</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSlider" name="sliderGain">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>180</y>
         <width>321</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbHighgain">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>191</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>设置探测器高增益</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbNonuniform">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>191</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>开启非均匀性校正</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbIntegral">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>201</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>开启自动帧积分时间调整</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>帧周期(ms)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>帧积分时间(ms)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAdjustOnsite">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>现场修正校正系数</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="editCycle">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>420</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="editIntegral">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>460</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSetCycle">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>420</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSetIntegral">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>460</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbHistogram">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>181</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>直方图均衡</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbSmooth">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>191</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>开启多帧平滑</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelip">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>68</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>IP地址：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>68</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>端口号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="portnum">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="IPCtrl" name="ipaddr">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="rightPanel">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <widget class="QGroupBox" name="dispGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>331</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>显示参数</string>
       </property>
       <widget class="QCheckBox" name="cbMirror">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>左右镜像</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbKeepRatio">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>保持长宽比</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>帧频（FPS）</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="fps">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>80</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>331</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>保存路径</string>
       </property>
       <widget class="QLabel" name="labelPath">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Desktop</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnBrowse">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btnConnect">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>连接相机</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPhoto">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>保存图片</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnVideo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>开始录像</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IPCtrl</class>
   <extends>QFrame</extends>
   <header>ipctrl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
